### PR TITLE
Copyright header and other cleanups

### PR DESCRIPTION
**Pull request by parthpp**
Description:
This patch adds copyright header information to source files. It also removes some commented code and `console.log` statements.

Linting:

- [x] No linting errors

Tests:

- [ ] Manual tests

Documentation:

- [ ] Requires documentation updates - NO

### CODE REVIEW

File: ep-lambda-function/handlers/getCart.handler.js [hunk 1]
```diff
@@ -21,4 +42,4 @@ const GetCartHandler = async function (intentRequest, callback, sessionCart) {
     {'contentType': 'PlainText', 'content': `${await getCartTotal(intentRequest)}`}));
 };
 
-module.exports = GetCartHandler;
+module.exports = GetCartHandler;
```
Comment (shaunmaharaj) [inline ep-lambda-function/handlers/getCart.handler.js]: Need a new line added here

File: ep-lambda-function/handlers/removeFromCart.handler.js [hunk 2]
```diff
@@ -26,4 +47,4 @@ const RemoveFromCartHandler = async function (intentRequest, callback) {
     {'contentType': 'PlainText', 'content': lexReply}));
 };
 
-module.exports = RemoveFromCartHandler;
+module.exports = RemoveFromCartHandler;
```
Comment (shaunmaharaj) [inline ep-lambda-function/handlers/removeFromCart.handler.js]: missing new line

File: ep-lambda-function/interceptors/auth.interceptor.js [hunk 3]
```diff
@@ -32,4 +32,4 @@ const AuthInterceptor = {
     }
 }
 
-module.exports = AuthInterceptor;
+module.exports = AuthInterceptor;
```
Comment (shaunmaharaj) [inline ep-lambda-function/interceptors/auth.interceptor.js]: missing new line

**Comment by parthpp** [comment]
@shaunmaharaj fixed review comments.

**Pull request merged**